### DISCUSSION TITLE
Set "en-GB" for "en" locale

### DISCUSSION THREAD

**Discussion by Hauglid** [amannn/next-intl#924] [Q&A]
Hi! I would like to use "en-GB" for "en" locale and not en-US. This is because of the date format. So the questions is really, how can I use "en-GB" date order with "en" locale.

**Comment by amannn** [comment]
This is planned, please see https://github.com/amannn/next-intl/issues/653!

**Comment by Hauglid** [comment]
Do you have an estimate for when this will be implemented? 

Also, can you please delete the duplicate (closed) question? I posted it from the wrong account.

**Reply by amannn** [reply]
Currently unfortunately no estimate, but there‘s a community-maintained implementation of this that‘s mentioned in the thread. Maybe that works for you!
